bucket/openapi-generator-cli.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "5.3.1",
+    "description": "OpenAPI Generator allows generation of API client libraries (SDK generation), server stubs, documentation and configuration automatically given an OpenAPI Spec (v2, v3).",
+    "homepage": "https://openapi-generator.tech/",
+    "license": "Apache-2.0",
+    "suggest": {
+        "JDK": "java/openjdk"
+    },
+    "url": "https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/5.3.1/openapi-generator-cli-5.3.1.jar#/openapi-generator-cli.jar",
+    "hash": "sha1:7b1671e987726e327483d7e0ef095e7f4aa834cd",
+    "pre_install": "'@java -jar %~dp0openapi-generator-cli.jar %*' | Out-File \"$dir\\openapi-generator-cli.cmd\" -Encoding ASCII",
+    "bin": "openapi-generator-cli.cmd",
```

```suggestion
    "bin": "openapi-generator-cli.jar",
```

No need to create file. See https://github.com/ScoopInstaller/Scoop/blob/5602083868ed04224ea8edf1ff0c3a2a3f6588b9/lib/core.ps1#L632-L640